### ISSUE TITLE
Lightning Talk: Techniques to Ensure User Privacy when Extracting Business Insights from Data

### ISSUE THREAD

**Issue by festusdrakon** [pydatadelhi/talks#117]
- Abstract (2-3 lines)
A walk through how differential privacy and federated learning are implemented to maintain individual user privacy even when the data is being used to generate insights on mass customer behavior to ensure targeted online advertising is implemented at scale.  

- Brief Description and Contents to be covered
How targeted advertising generally works
The need for customer privacy in generating insights for advertising
Differential Privacy as a technique
Federated Learning as another effective technique
A brief introduction to PySyft for decoupling data for Federated Learning 

- Pre-requisites for the talk
A very basic familiarity with elementary data analysis workflows, and predictive modeling.

- Time required for the talk
12 minutes 

- Link to slides
Will share within 48-72 hours upon the acceptance of the talk proposal.

- Will you be doing hands-on demo as well?
No

- Link to ipython notebook (if any)
None

- About yourself
I have past experience with strategizing and implementing digital marketing campaigns for small scale startups, and so have been at that end of the pipeline where we were required to draw business insights from standard website metrics to optimize performance of advertising campaigns and deliver targeted services to our intended audience. I have also worked as a content creator and truly understand the value of privacy and strive always to fiercely protect it. As Udacity's Secure and Private AI Challenge Scholar I was educated about the technical latticework behind ensuring individual privacy, and look forward to sharing that knowledge with the extended developer ecosystem at large.  

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?
Yes

- Any query ?
None

**Comment by sanketverma1704** [comment]
Hi @festusdrakon.
Can you provide us with the slides when they're done as the reviewer needs to go through them while reviewing the proposal?

**Comment by festusdrakon** [comment]
Yes, of course. I'll upload the slides here by this time tomorrow, once I'm through with a couple of pressing prior commitments, if that's alright?

**Comment by sanketverma1704** [comment]
Yeah, sure. :)
